crates/sui-indexer-alt-graphql/src/api/types/transaction/mod.rs

```diff
@@ -253,18 +272,82 @@ impl Transaction {
     }
 }
 
+async fn tx_affected_address(
+    ctx: &Context<'_>,
+    Range {
+        start: tx_lo,
+        end: tx_hi,
+    }: Range<u64>,
+    page: &Page<CTransaction>,
+    SuiAddress(affected_address): SuiAddress,
+) -> Result<Vec<u64>, RpcError> {
+    let pg_reader: &PgReader = ctx.data()?;
+
+    let is_asc = page.is_from_front();
+    let query = query!(
+        r#"
+SELECT
+    tx_sequence_number
+FROM
+    tx_affected_addresses
+WHERE
+    {BigInt} <= tx_sequence_number /* tx_lo */
+    AND tx_sequence_number < {BigInt} /* tx_hi */
+    AND affected = {Bytea} /* affected_address */
+ORDER BY
+    CASE WHEN {Bool} /* is_asc */ = true THEN tx_sequence_number ELSE NULL END ASC,
+    CASE WHEN {Bool} /* is_asc */ = false THEN tx_sequence_number ELSE NULL END DESC
```

Comment:
TIL that you could even do this, but it may be neater to do this in Rust, and interpolate in a query snippet accordingly, like so:

https://github.com/MystenLabs/sui/blob/d15186fe182494b04971ae728801f36ec7968788/crates/sui-indexer-alt-graphql/src/api/types/event/mod.rs#L152-L173